frontend/src/folder/pages/EditFolder.tsx

```diff
@@ -0,0 +1,70 @@
+import { BsThreeDots } from 'react-icons/bs';
+import useFolderActions from '../hooks/useFolder';
+import { useFolderContext } from '../hooks/useFolderContext';
+
+export default function EditFolder() {
+  const {
+    newTitle,
+    setNewTitle,
+    isEditing,
+    handleEditClick,
+    showOptions,
+    handleCancelClick,
+    handleSaveClick,
+    toggleOptions,
+    optionsRef,
+    inputRef,
+    handleDeleteClick,
+  } = useFolderActions();
+  const { selectedFolderTitle } = useFolderContext();
+
+  return (
+    <div className="flex justify-between items-center border-b py-2 px-2">
+      {isEditing ? (
+        <input
```

Comment:
Use shadcn input instead

You can create a new variant in the config for it to make it look exactly this way

Reply:
Done!